inc/massiveaction.class.php

```diff
@@ -881,6 +881,25 @@ static function showMassiveActionsSubForm(MassiveAction $ma) {
                   $values = $ma->POST['additionalvalues'];
                }
                $values[$search["field"]] = '';
+               if( $fieldname == 'itilcategories_id' ){
+                  $condition='';
+                  switch( $item->getType() ) {
+                     case 'Ticket':
+                        $condition = '`is_incident` = 1 AND `is_request` = 1' ;
+                        break ;
+                     case 'Problem':
+                        $condition = '`is_problem` = 1' ;
+                        break ;
+                     case 'Change':
+                        $condition = '`is_change` = 1' ;
+                        break ;
+                  }
+                  if( isset( $search['condition'] ) ){
+                     $search['condition'] .= ' AND '.$condition ;
+                  } else {
+                     $search['condition'] = $condition ;
+                  }
```

Comment:
and categories have a life... one can be active at a time, and it can be de-activated later, but we need to continue to search for it, but don't want to assign it anymore